src/nrc/tests/setup_configuration/test_abonnementen_configuration.py

```diff
@@ -0,0 +1,156 @@
+from django.test import TestCase
+
+from django_setup_configuration.exceptions import ConfigurationRunFailed
+from django_setup_configuration.test_utils import execute_single_step
+
+from nrc.datamodel.models import Abonnement, Filter, FilterGroup, Kanaal
+from nrc.setup_configuration.abonnementen import AbonnementConfigurationStep
+
+CONFIG_FILE_PATH = (
+    "src/nrc/tests/setup_configuration/files/setup_config_abonnementen_config.yaml"
+)
+CONFIG_FILE_PATH_EMPTY_KANALEN = "src/nrc/tests/setup_configuration/files/setup_config_abonnementen_empty_kanalen.yaml"
+
+
+class AbonnementenConfigurationTests(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.kanaal_zaken = Kanaal.objects.create(
+            naam="zaken",
+            documentatie_link="http://localhost:8000/old-documentation/zaken",
+            filters=["zaaktype", "vertrouwelijkheidaanduiding"],
+        )
+        self.kanaal_documenten = Kanaal.objects.create(
+            naam="documenten",
+            documentatie_link="http://localhost:8000/old-documentation/zaken",
+            filters=["informatieobjecttype", "vertrouwelijkheidaanduiding"],
+        )
```

Comment:
I think defining these in the tests that need them makes the tests more explicit and removes the need to call `self.kanaal_zaken.delete()` for some

Reply:
I'm always torn here, on the one hand I agree, but it would mean this is duplicated 5 times (and possibly more if more tests are added), so I think I'd rather keep it in setUp